doc/man-template.roff

```diff
@@ -1,45 +1,145 @@
-[OPTIONS]
-.SS "Positional Arguments"
-.HP
-METHOD
-.IP
-The HTTP method to use. One of \fBget\fR, \fBpost\fR, \fBhead\fR, \fBput\fR, \fBpatch\fR, \fBdelete\fR, or \fBoptions\fR.
-.IP
-Optional. If omitted, a GET or a POST will be done depending on whether the request sends data.
-.HP
-URL
-.IP
-The URL to request. The scheme defaults to 'http://' unless \fB\-\-https\fR is used.
-.IP
-A leading colon works as shorthand for localhost. :8000 is equivalent to localhost:8000, and :/path is equivalent to localhost/path.
-.HP
-REQUEST_ITEM...
-.IP
-Optional key-value pairs to be included in the request.
-.IP
-\fBkey==value\fR to add a parameter to the URL
-.IP
-\fBkey=value\fR to add a JSON field (\fB\-\-json\fR) or form field (\fB\-\-form\fR)
-.IP
-\fBkey=@value\fR to add a JSON field (\fB\-\-json\fR) or form field (\fB\-\-form\fR) from a file.
-.IP
-\fBkey:=value\fR to add a complex JSON value (e.g. `numbers:=[1,2,3]`)
-.IP
-\fBkey:=@value\fR to add a complex JSON value (e.g. `numbers:=@data.json`) from a file.
-.IP
-\fBkey@filename\fR to upload a file from filename (with \fB\-\-form\fR)
-.IP
-\fB@filename\fR to use a file as the request body
-.IP
-\fBheader:value\fR to add a header
-.IP
-\fBheader:\fR to unset a header
-.IP
-\fBheader;\fR to add a header with an empty value
-.IP
-A backslash can be used to escape special characters (e.g. weird\\:key=value).
-.IP
-File uploads can take ';type=' and ';filename=' parameters to set the mimetype and filename, respectively. For example:
-.IP
-  key@filename;type=text/html;filename=document.html
-.SS "Options"
+.TH XH 1 {{date}} {{version}} "User Commands"
+
+.SH NAME
+xh \- Friendly and fast tool for sending HTTP requests
+
+.SH SYNOPSIS
+.B xh
+[\fIOPTIONS\fR]
+[\fIMETHOD\fR]
+\fIURL\fR
+[\-\-\]
+[\fIREQUEST_ITEM\fR ...]
+
+.SH DESCRIPTION
+
+xh is an HTTP client that aims to reimplement as much as possible of
+HTTPie's excellent design, with a focus on improved performance.
+
+xh uses HTTPie's request-item syntax to set headers, request body,
+query string, etc. For more details, refer to REQUEST_ITEM in the
+OPTIONS section.
```

Comment:
The fact that (like most features) it's copied from HTTPie isn't directly relevant. Maybe this:
```suggestion
Headers, request body, query string, etcetera can be set using
positional arguments. For more details, refer to REQUEST_ITEM in the
OPTIONS section.
```
Or maybe just remove the whole paragraph.

Reply:
I've opted to update it, but I wouldn't mind removing it and the paragraph after it.